### DISCUSSION TITLE
How'd I set the default color cycle in a theme in the current version of plotnine?

### DISCUSSION THREAD

**Discussion by andyljones** [has2k1/plotnine#645] [Q&A]
As of a few versions back, I could set the color cycle with

```
class MyTheme(pn.theme):

    def __init__(self):
        super().__init__(complete=True)

        self._rcParams.update({'axes.prop_cycle': my_cycler})
```

I've updated plotnine though and (a) I have to turn `complete` off now, as it seems to be a heavier requirement than it used to be and (b) with it off, `axes.prop_cycle` doesn't seem to take effect. 

How'd I set the same rcParam entry now?

Thanks!

**Comment by has2k1** [comment]
I am surprised that the `prop_cycle` could have ever worked! I cannot see how. Matplotlib `prop_cycle` conflicts with the default aesthetic values (when there is no mapping to the aesthetic) or the discrete scales (when there is a mapping to the aesthetic).

When creating a theme, it is better to inherit from one of the provided themes and alter as you please. Otherwise you have to specify a bunch of probably uninteresting little details. Previously the code had some embedded values to fallback on, that is not the case anymore they all must come from the theme.

**Comment by andyljones** [comment]
Thanks! Can you point me at a provided theme that alters the default aesthetics? I had a look at [seaborn](https://github.com/has2k1/plotnine/blob/ad878c21f09de962cfbd166d502072673b4404a9/plotnine/themes/theme_seaborn.py) and I couldn't spot it there. [This module](https://github.com/has2k1/plotnine/blob/80ff224d43012dd7e46264fea7860d42899a207e/plotnine/themes/seaborn_rcmod.py) has a bunch of color-code related docs but those args seem to have been removed.
